### DISCUSSION TITLE
What is 'integrable' for?

### DISCUSSION THREAD

**Discussion by bcalco** [leppie/IronScheme#116] [Discussions] [answered]
@leppie Could use some explanation what the `(ironscheme integrable)` library is for. Perhaps I have had insufficient quantities of coffee this morning; or perhaps I have had too much. Either way, it's not jumping out at me what it's for, or how to use it. I am actually hoping it's one of those "duh, of course, how could I miss that?" moments. I went on an admittedly not exhaustive search for IronScheme libs that use it, but didn't find any before my impatience got the better of me. So I decided to log a quick request here for information.

**Comment by leppie** [comment, marked as answer]
See https://www.scheme.com/tspl4/syntax.html halfway down the examples at the end.

Basically it will inline the code.

**Reply by bcalco** [reply]
I had a sneaking suspicion I should have understood it. LOL. "Duh, of course, how could I miss that?" :)

**Reply by lassik** [reply]
MIT Scheme has `define-integrable` built in.
